charts/application/values.yaml

```
@@ -141,6 +141,10 @@ serviceAccount:
   mountPath: /config/service-account
   # k8s ServiceAccount.automountServiceAccountToken setting
   automountServiceAccountToken: false
+  # gives the application the defined role binding
+  rbac:
+    roleBindings: []
+    clusterRoleBindings: []
```

```suggestion
  rbac:
    roleBindings: []
    #  - roleType: ClusterRole
    #    roleName: admin
    clusterRoleBindings: []
    #  - roleType: ClusterRole
    #    roleName: edit
```

so that you don't have to check the actual template file to see what's possible.

---

Would <3 a way to also have that in the helm-docs